platform/core/src/services/HangingProtocolService/HangingProtocolService.ts

```diff
@@ -524,6 +657,7 @@ class HangingProtocolService {
         const seriesMatchDetails = this.protocolEngine.findMatch(
           displaySet,
           seriesMatchingRules,
+          // Todo: why we have images here since the matching type does not have it
```

Comment:
The reason to have images here is to allow getting the metadata to read attributes not included in the study by default - there are other possible patient/study level attributes.